units/ArmBuildings/LandDefenceOffence/armamb.lua

```diff
@@ -139,36 +137,73 @@ return {
 					subs = 150,
 					vtol = 90,
 				},
+				customparams = {
+					exclude_preaim = true,
+					smart_preferred_weapon = true,
+				}
 			},
 			armamb_gun_high = {
 				accuracy = 400,
-				areaofeffect = 224,
+				areaofeffect = 152,
```

Comment:
If most of these values are the same across multiple weapons in the unit then perhaps copy them over from a single "master" table. Perhaps the balance parts of this could also be extracted into a separate PR and merged more quickly (leaving this PR with just the removal of the high traj toggle).

Reply:
If we start using procedural code and/or comments in def files, we have to remove def baking, or completely rework it.

Reply:
Sounds to me like outside the scope of this PR for the time being.